packages/browser-repl/src/lib/autocompleter/autocompleter.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
It's a bit weird for me to have a `./lib` directory within a `./src`, since lib is usually where our compiled modules go. Maybe adding this to `./modules` might work better?

Reply:
You mean rename `lib`? Sure, i think another option would be to drop `lib` and put everything as peer of components.

Reply:
> i think another option would be to drop lib and put everything as siblings of components.

that's also a good idea!